pkg/agent/datamodel/sig_config.go

```diff
@@ -12,6 +15,12 @@ const (
 	AzurePublicCloudSigSubscription string = "109a5e88-712a-48ae-9078-9ca8b3c81345" // AKS VHD
 )
 
+func init() {
+	_, filename, _, _ := runtime.Caller(0)
+	manifestFilePath := "../../../parts/linux/cloud-init/artifacts/manifest.json"
+	getCachedK8sVersionFromManifest(path.Join(path.Dir(filename), manifestFilePath))
```

Comment:
When you run `go test` it will calculate the relative file path from where you call go test. 

manifestFilePath worked as intended when called inside `pkg/agent/` but it would fail if you ran `go test` in `pkg/agent/datamodel`. So the `_, filename, _, _ := runtime.Caller(0)` is a workaround to make sure the relative path is from within sig_config.go